todoapp/app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/TasksLocalDataSourceTest.java

```diff
@@ -79,13 +85,13 @@ public void saveTask_retrievesTask() {
         mLocalDataSource.saveTask(newTask);
 
         // Then the task can be retrieved from the persistent repository
-        mLocalDataSource.getTask(newTask.getId());
+        TestSubscriber<Task> testSubscriber = new TestSubscriber<>();
```

Comment:
`TestSubscriber<Task>` is actually used only in 2 tests out of 17. I wouldn't do a member just for that.
I'll do one for `TestSubscriber<List<Task>>` since that's used for more tests.